lib/vanagon/platform/windows.rb

```diff
@@ -56,6 +56,16 @@ def generate_msi_packaging_artifacts(workdir, name, binding)
         FileUtils.mkdir_p(File.join(workdir, "wix"))
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.wxs.erb"), File.join(workdir, "wix",  "#{name}.wxs"), false, { :binding => binding })
         erb_file(File.join(VANAGON_ROOT, "resources/windows/wix/project.filter.xslt.erb"), File.join(workdir, "wix", "#{name}.filter.xslt"), false, { :binding => binding })
+
+        # Project Specific Wix Files (ERB Templates)
+        ["customactions", "properties", "sequences"].each do |wix_filename|
+          erb_file("resources/windows/wix/#{wix_filename}.wxs.erb", File.join(workdir, "wix",  "#{wix_filename}.wxs"), false, { :binding => binding })
```

Comment:
Based on @melissa's comments on the other PR, it looks like we might also want to recognize non-erb `.wxs` files and copy them as well.